autoload/codefmt.vim

```diff
@@ -98,18 +95,20 @@ function! codefmt#EnsureFormatter(formatter) abort
 endfunction
 
 
+""
+" @private
 " Formatter: js-beautify
-if !exists('s:js_beautify')
-  let s:js_beautify = {
+function! codefmt#GetJsBeautifyFormatter() abort
+  let l:formatter = {
       \ 'name': 'js-beautify',
       \ 'setup_instructions': 'Install js-beautify ' .
           \ '(https://www.npmjs.com/package/js-beautify).'}
 
-  function s:js_beautify.IsAvailable() abort
+  function l:formatter.IsAvailable() abort
```

Comment:
I hadn't seen this syntax at all before, so I just made the change optimistically.  It seems to work; as you say, if we find out that it doesn't work in some cases, we can easily switch the syntax.